src/app/user/shared/component/pre/pre.component.ts

```diff
@@ -0,0 +1,24 @@
+import {Component, OnInit} from '@angular/core';
+import {ActivatedRoute} from '@angular/router';
+
+@Component({
+  selector: 'app-pre',
+  templateUrl: './pre.component.html',
+  styleUrls: ['./pre.component.scss']
+})
+export class PreComponent implements OnInit {
```

Comment:
would you chose better name for the component , after 6 months you will not remember this